Automattic/jetpack
# [Super cache] Fix a bunch of linting issues

This PR fixes a bunch of linting warnings. It aims to avoid changes to functionality while cleaning out warnings. There are more to go, but this is a good start.

#### Changes proposed in this Pull Request:
* Fix a bunch of linting warnings.

#### Other information:

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Run the test suite to ensure it all works.
* Try the plugin out.
* Read the code.
